mlir/tools/miopen-gen/miopen-gen.cpp

```
@@ -314,9 +321,10 @@ static cl::opt<std::string> randomSeed(
     "rand",
     cl::desc(
         "A positive integer or zero indicates the seed of random data generator"
-        "for convolution inputs, e.g. -rand 1. If not specifed, or -rand none, "
-        "use all ones. If 0, use time(0) as the seed."),
-    cl::value_desc("seed"), cl::init("none"));
+        "for convolution inputs, e.g. -rand 1. If not specifed, or 'fixed', "
+        "use a fixed nonuniform test pattern. If 'none', use all 1s as the "
+        "values. If 0, use time(0) as the seed."),
+    cl::value_desc("seed"), cl::init("fixed"));
```

Good move to default to random vectors.